ccd-definition/CaseEventToFields/CareSupervision/uploadDocuments.json

```diff
@@ -143,6 +154,7 @@
     "DisplayContext": "READONLY",
     "PageID": 1,
     "PageDisplayOrder": 1,
-    "PageColumnNumber": 1
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Y"
```

Comment:
Or this

Reply:
unless they are there for come formatting of the CYA

Reply:
no looks better without so removed!